### DISCUSSION TITLE
Nginx working config?

### DISCUSSION THREAD

**Discussion by demogorgonz** [postalserver/postal#2170] [Help with using Postal] [answered]
Does anyone have working nginx config?

**Comment by demogorgonz** [comment, marked as answer]
Made it working:

```
server {
   listen 443 ssl;
    ssl_certificate     /path/cert.pem;
    ssl_certificate_key /path/privkey.pem;
  server_name postal.example.com;

  location / {
       proxy_pass http://127.0.0.1:5000;
       proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
       proxy_set_header Host $http_host;
       proxy_set_header  X-Forwarded-Proto $scheme;
       proxy_set_header  X-Forwarded-Ssl on;
       proxy_set_header  X-Forwarded-Port $server_port;
       proxy_set_header  X-Forwarded-Host $host;
       proxy_redirect off;
  }
}

```

**Reply by willpower232** [reply]
glad you figured it out, there is a more detailed example over in https://github.com/postalserver/postal/discussions/2012#discussioncomment-2889283

**Reply by demogorgonz** [reply]
Haven't saw that issue, shouldn't this be in docs ? Thanks

**Reply by demogorgonz** [reply]
@willpower232 i can't find anything related to `Click & Open tracking` for nginx, do you know of any reference maybe ? Thank you in advance.

**Reply by willpower232** [reply]
As per the documentation you need to add a header to the proxy https://docs.postalserver.io/features/click-and-open-tracking

So you can duplicate the server block you already have for the tracking domain and then add `proxy_set_header X-Postal-Track-Host 1;`

**Reply by demogorgonz** [reply]
That's it, i had a typo :) thank you 👍
